src/containerapp/azext_containerapp/custom.py

```diff
@@ -2068,6 +2068,9 @@ def containerapp_up_logic(cmd, resource_group_name, name, managed_env, image, en
     if containerapp_def:
         ca_exists = True
 
+    if image is None:
+        image = "mcr.microsoft.com/azuredocs/containerapps-helloworld:latest"
```

Comment:
NIT: Can we add comment explaining the need for this - so we don't accidentally delete / remove this?